fairscale/experimental/nn/ssd_offload.py

```diff
@@ -9,10 +9,13 @@
 from functools import reduce
 import io
 import os
-from typing import Any, Iterator, List, Optional, Sequence, Tuple, Type
+import pickle
+from types import TracebackType
+from typing import IO, Any, BinaryIO, Iterator, List, Optional, Sequence, Tuple, Type, Union
 
 import numpy as np
 import torch
+from torch.serialization import DEFAULT_PROTOCOL as DEFAULT_PROTOCOL
```

Comment:
do we need the "as" since DEFAULT_PROTOCOL is the same? maybe i am missing something about Python?

Reply:
The "as" allows us to use "DEFAULT_PROTOCOL" instead of "torch.serialization.DEFAULT_PROTOCOL" later in the file.